helm/tests/storage/pvc_test.yaml

```
@@ -0,0 +1,84 @@
+suite: "Storage PVC Tests"
+
+templates:
+  - "templates/storage/pvc.yaml"
+
+tests:
+  - it: "should render a PVC with custom size, access mode and storage class"
+    set:
+      persistence:
+        enabled: true
+        storageData:
+          enabled: true
+          size: 10Gi
+          accessMode: ReadWriteMany
+          storageClass: sqlite-storageclass-test
+    asserts:
+      - equal:
+          path: "metadata.name"
+          value: "RELEASE-NAME-sbombastic-storage-data"
+      - equal:
+          path: "spec.storageClassName"
+          value: "sqlite-storageclass-test"
+      - equal:
+          path: "spec.accessModes"
+          value:
+          - ReadWriteMany
+      - equal:
+          path: "spec.resources.requests.storage"
+          value: "10Gi"
+  - it: "should use the storage class defined by the user"
+    set:
+      persistence:
+        enabled: true
+        storageData:
+          enabled: true
+          storageClass: "fast-storageclass-test"
+    asserts:
+      - equal:
+          path: "spec.storageClassName"
+          value: "fast-storageclass-test"
+  - it: "should use allow usage of an empty string storage class to allow cluster's default one to be used"
```

sorry, I made a typo:

```suggestion
  - it: "should allow the usage of an empty string storage class to allow cluster's default one to be used"
``